### PR TITLE
i18n(zh-cn): update `guides/content-collections.mdx`

### PR DESCRIPTION
**Pull request by 100gle**
#### Description (required)

update `guides/content-collections.mdx`

#### Related issues & labels (optional)

#5045

### CODE REVIEW

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 1]
```diff
@@ -238,12 +238,12 @@ export const collections = {
 
 ### 用 Zod 定义数据类型
 
-Astro 使用[Zod](https://github.com/colinhacks/Zod)驱动其内容模式。Astro 使用 Zod 能够验证集合中每个文件的frontmatter，并在从项目内部查询内容时提供自动的 TypeScript 类型。
+Astro 使用[Zod](https://github.com/colinhacks/Zod)驱动其内容模式。Astro 使用 Zod 能够验证集合中每个文件的 frontmatter，并在从项目内部查询内容时提供自动的 TypeScript 类型。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:241]:
```suggestion
Astro 使用 [Zod](https://github.com/colinhacks/Zod) 驱动其内容模式。Astro 使用 Zod 能够验证集合中每个文件的 frontmatter，并在从项目内部查询内容时提供自动的 TypeScript 类型。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 2]
```diff
@@ -238,12 +238,12 @@ export const collections = {
...
 
 要在 Astro 中使用 Zod，请从 `"Astro:content"`中导入 `z`。这是 Zod 库的重新导出，它支持 Zod 的所有特性。查看[Zod 的 README](https://github.com/colinhacks/Zod)获得关于 Zod 如何工作以及可用特性的完整文档。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:243]:
```suggestion
要在 Astro 中使用 Zod，请从 `"Astro:content"` 中导入 `z`。这是 Zod 库的重新导出，它支持 Zod 的所有特性。查看[Zod 的 README](https://github.com/colinhacks/Zod)获得关于 Zod 如何工作以及可用特性的完整文档。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 3]
```diff
@@ -419,7 +419,7 @@ const englishDocsEntries = await getCollection('docs', ({ id }) => {
 
 ### 使用 Astro 模板中的内容
 
-查询完集合条目后，可以直接在 Astro 组件模板内访问每个条目。这使你可以呈现诸如内容链接(使用内容 `slug` )或关于内容的信息(使用 `data` 属性)之类的 HTML。
+查询完集合条目后，可以直接在 Astro 组件模板内访问每个条目。这使你可以呈现诸如内容链接 (使用内容 `slug` ) 或关于内容的信息 (使用 `data` 属性) 之类的 HTML。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:422]:
```suggestion
查询完集合条目后，可以直接在 Astro 组件模板内访问每个条目。这使你可以呈现诸如内容链接 (使用内容 `slug`) 或关于内容的信息 (使用 `data` 属性) 之类的 HTML。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 4]
```diff
@@ -478,13 +478,13 @@ const { Content, headings } = await entry.render();
 
 ## 从内容生成路由
 
-内容集合存储在 `src/pages/` 目录之外。这意味着默认情况下不为集合项生成路由。你将需要手动创建一个新的[动态路由](/zh-cn/core-concepts/routing/#动态路由) ，以便从集合条目生成 HTML 页面。动态路由将映射传入的请求参数(例如: 在 `src/pages/blog/[...slug].astro` 中的 `Astro.params.slug`)获取集合中的正确条目。
+内容集合存储在 `src/pages/` 目录之外。这意味着默认情况下不为集合项生成路由。你将需要手动创建一个新的[动态路由](/zh-cn/core-concepts/routing/#动态路由) ，以便从集合条目生成 HTML 页面。动态路由将映射传入的请求参数 (例如：在 `src/pages/blog/[...slug].astro` 中的 `Astro.params.slug`) 获取集合中的正确条目。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:481]:
```suggestion
内容集合存储在 `src/pages/` 目录之外。这意味着默认情况下不为集合项生成路由。你将需要手动创建一个新的[动态路由](/zh-cn/core-concepts/routing/#动态路由)，以便从集合条目生成 HTML 页面。动态路由将映射传入的请求参数 (例如：在 `src/pages/blog/[...slug].astro` 中的 `Astro.params.slug`) 获取集合中的正确条目。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 5]
```diff
@@ -478,13 +478,13 @@ const { Content, headings } = await entry.render();
...
 
-生成路由的确切方法将取决于你的构建[`output`](/zh-cn/reference/configuration-reference/#output)模式: `static`(默认值)或 `server`(针对 SSR)。
+生成路由的确切方法将取决于你的构建[`output`](/zh-cn/reference/configuration-reference/#output)模式：`static`(默认值) 或 `server`(针对 SSR)。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:483]:
```suggestion
生成路由的确切方法将取决于你的构建 [`output`](/zh-cn/reference/configuration-reference/#output) 模式：`static` (默认值) 或 `server` (针对 SSR)。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 6]
```diff
@@ -478,13 +478,13 @@ const { Content, headings } = await entry.render();
...
 
-### 构建为静态输出(默认)
+### 构建为静态输出 (默认)
 
-如果你正在构建一个静态网站(Astro 的默认行为) ，你可以使用[`getStaticPaths()`](/zh-cn/reference/api-reference/#getstaticpaths)函数在构建过程中从一个 `src/pages/` 组件创建多个页面。
+如果你正在构建一个静态网站 (Astro 的默认行为) ，你可以使用[`getStaticPaths()`](/zh-cn/reference/api-reference/#getstaticpaths)函数在构建过程中从一个 `src/pages/` 组件创建多个页面。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:487]:
```suggestion
如果你正在构建一个静态网站 (Astro 的默认行为) ，你可以使用 [`getStaticPaths()`](/zh-cn/reference/api-reference/#getstaticpaths) 函数在构建过程中从一个 `src/pages/` 组件创建多个页面。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 7]
```diff
@@ -478,13 +478,13 @@ const { Content, headings } = await entry.render();
...
 
 调用 `getStaticPaths()` 中的[`getCollection()`](/zh-cn/reference/api-reference/#getcollection)来查询内容。然后，使用每个内容条目的 `slug` 属性创建新的 URL 路径。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:489]:
```suggestion
调用 `getStaticPaths()` 中的 [`getCollection()`](/zh-cn/reference/api-reference/#getcollection) 来查询内容。然后，使用每个内容条目的 `slug` 属性创建新的 URL 路径。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 8]
```diff
@@ -513,9 +513,9 @@ const { Content } = await entry.render();
 如果你的自定义 slug 包含 `/` 字符来生成具有多个路径段的 URL，则必须使用 [rest 参数 (`[...slug]`)](/zh-cn/core-concepts/routing/#剩余参数)在此动态路由页面的 `.astro` 文件名中。
 :::
 
-### 构建为服务器输出(SSR)
+### 构建为服务器输出 (SSR)
 
-如果你正在构建一个动态网站(使用 Astro 的 SSR 支持) ，则不需要在构建期间提前生成任何路径。相反，你的页面应该检查请求(使用 `Astro.request` 或 `Astro.params` )以按需找到 `slug` ，然后使用[`getEntry()`](/zh-cn/reference/api-reference/#getentry)获取它。
+如果你正在构建一个动态网站 (使用 Astro 的 SSR 支持) ，则不需要在构建期间提前生成任何路径。相反，你的页面应该检查请求 (使用 `Astro.request` 或 `Astro.params` ) 以按需找到 `slug` ，然后使用[`getEntry()`](/zh-cn/reference/api-reference/#getentry)获取它。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:518]:
```suggestion
如果你正在构建一个动态网站 (使用 Astro 的 SSR 支持)，则不需要在构建期间提前生成任何路径。相反，你的页面应该检查请求 (使用 `Astro.request` 或 `Astro.params`) 以按需找到 `slug`，然后使用 [`getEntry()`](/zh-cn/reference/api-reference/#getentry) 获取它。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 9]
```diff
@@ -546,7 +546,7 @@ const { Content } = await entry.render();
 
 2. 为内容集合[设置 TypeScript ](#设置-typescript)。
 
-3. 创建至少一个集合(`src/content/` 的文件夹中) ，并将 Markdown 和 MDX 页面从 `src/pages/` 移动到 `src/content/` 的这些子目录中。当同一集合中的所有文件具有相似的 frontmatter 属性时，集合的工作效果最好。因此，请选择新的文件夹结构以反映类似类型的页面。
+3. 创建至少一个集合 (`src/content/` 的文件夹中) ，并将 Markdown 和 MDX 页面从 `src/pages/` 移动到 `src/content/` 的这些子目录中。当同一集合中的所有文件具有相似的 frontmatter 属性时，集合的工作效果最好。因此，请选择新的文件夹结构以反映类似类型的页面。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:549]:
```suggestion
3. 创建至少一个集合 (`src/content/` 的文件夹中)，并将 Markdown 和 MDX 页面从 `src/pages/` 移动到 `src/content/` 的这些子目录中。当同一集合中的所有文件具有相似的 frontmatter 属性时，集合的工作效果最好。因此，请选择新的文件夹结构以反映类似类型的页面。
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 10]
```diff
@@ -620,7 +620,7 @@ const { Content } = await entry.render();
 
 7. 将 `Astro.glob()` 替换为[`getCollection()`](/zh-cn/reference/api-reference/#getcollection)以从 Markdown 文件中获取内容和元数据。还需要更新对返回的 post 对象的引用，因为你现在可以在 `data` 属性上找到 frontmatter 值。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/content-collections.mdx:621]:
```suggestion
7. 将 `Astro.glob()` 替换为 [`getCollection()`](/zh-cn/reference/api-reference/#getcollection) 以从 Markdown 文件中获取内容和元数据。还需要更新对返回的 post 对象的引用，因为你现在可以在 `data` 属性上找到 frontmatter 值。
```

**Review by Genteure — APPROVED** [review]
LGTM
All suggested changes are formatting changes.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**